docs/source/babs-init.rst

```diff
@@ -1,9 +1,53 @@
-*****************************************
+##################################################
 ``babs-init``: Initialize a BABS project
-*****************************************
+##################################################
+
+.. contents:: Table of Contents
+
+**********************
+Command-Line Arguments
+**********************
 
 .. argparse::
    :ref: babs.cli.babs_init_cli
    :prog: babs-init
-   :nodefault:
-   :nodefaultconst:
\ No newline at end of file
+
+    --input : @after
```

Comment:
```suggestion
   --input : @after
```

It turns out the problem was the indentation.

Reply:
Aha! You're right! Thank you so much Taylor!!! That's really hard to catch!!!